dvc/repo/add.py

```diff
@@ -49,7 +52,21 @@ def add(repo, targets, recursive=False, no_commit=False, fname=None):
 
             stages = _create_stages(repo, sub_targets, fname, pbar=pbar)
 
-            repo.check_modified_graph(stages)
+            try:
+                repo.check_modified_graph(stages)
+            except OverlappingOutputPathsError as e:
```

Comment:
```suggestion
            except OverlappingOutputPathsError as exc:
```
DS and others complain about such naming.